queries.sql

```diff
@@ -1,11 +1,62 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * FROM Animals WHERE name LIKE '%mon'; /*Find all animals whose name ends in "mon".*/
-SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-10' AND '2019-01-01' /*List the name of all animals born between 2016 and 2019.*/
-SELECT name FROM animals WHERE neutered='t' AND escape_attempts < 3; /*List the name of all animals that are neutered and have less than 3 escape attempts.*/
-SELECT date_of_birth from Animals WHERE name='Agumon' OR name ='Pikachu'; /* List date of birth of all animals named either "Agumon" or "Pikachu".*/
-SELECT name,escape_attempts FROM Animals WHERE weight_kg > 10.5; /* List name and escape attempts of animals that weigh more than 10.5kg */
-SELECT * FROM Animals WHERE neutered = 't'; /* Find all animals that are neutered.*/
-SELECT * FROM Animals WHERE  NOT name='Gabumon'; /* Find all animals not named Gabumon.*/
-SELECT * FROM Animals WHERE weight_kg >= 10.4 AND weight_kg <=17.3; /* Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)*/
+SELECT * FROM Animals WHERE name LIKE '%mon'; /*Find all Animals whose name ends in "mon".*/
+SELECT name FROM Animals WHERE date_of_birth BETWEEN '2016-01-10' AND '2019-01-01' /*List the name of all Animals born between 2016 and 2019.*/
+SELECT name FROM Animals WHERE neutered='t' AND escape_attempts < 3; /*List the name of all Animals that are neutered and have less than 3 escape attempts.*/
+SELECT date_of_birth from Animals WHERE name='Agumon' OR name ='Pikachu'; /* List date of birth of all Animals named either "Agumon" or "Pikachu".*/
+SELECT name,escape_attempts FROM Animals WHERE weight_kg > 10.5; /* List name and escape attempts of Animals that weigh more than 10.5kg */
+SELECT * FROM Animals WHERE neutered = 't'; /* Find all Animals that are neutered.*/
+SELECT * FROM Animals WHERE  NOT name='Gabumon'; /* Find all Animals not named Gabumon.*/
+SELECT * FROM Animals WHERE weight_kg >= 10.4 AND weight_kg <=17.3; /* Find all Animals with a weight between 10.4kg and 17.3kg (including the Animals with the weights that equals precisely 10.4kg or 17.3kg)*/
 
+
```

Comment:
Well done adding the missing transactions 👏🏾 
You are missing one more transition highlighted below as per project requirements.
![image](https://user-images.githubusercontent.com/38088818/171876926-8334eee9-716f-4df6-aaa8-8dc512cd2441.png)

For example:
```
   BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone
```

Reply:
Thank you for pointing this out. Updated!